### ISSUE TITLE
localePath not working after upgrade to 0.38.0+

### ISSUE THREAD

**Issue by vimutti77** [isaachinman/next-i18next#297]
## Describe the bug
After upgrade next-i18next to version 0.38.0+, if I set localePath to `public/locales`. the language will not be loaded on the server side.

## Occurs in next-i18next version
0.38.0 - 0.39.0

## Steps to reproduce
i18n.js
```
const NextI18Next = require('next-i18next/dist/commonjs')

module.exports = new NextI18Next({
  defaultLanguage: 'th',
  ignoreRoutes: ['/_next', '/static', '/public', '/app', '/icons', '/images'],
  otherLanguages: ['en'],
  localePath: 'public/locales',
  localeSubpaths: 'all',
})
```
My project structure
```
.
├── public
|     └── locales
└── src
      └── server.js
```

## Expected behaviour
The language are loaded correctly on server side in version 0.38.0.

## OS (please complete the following information)
 - Device: PC, Windows 10
 - Browser: Chrome lastest

## Additional context
The previous version (0.37.0) is working correctly.

**Comment by vimutti77** [comment]
I found that if I replicate the folder `public/locales` to `static/locales`, the language will be loaded. Look like the server is still load the language from `static/locales` although I set `localePath` to `public/locales`.

**Comment by novascreen** [comment]
I believe this happened here: https://github.com/isaachinman/next-i18next/commit/56363e4e3cf7c89f55675430efe7a251a7207eaa
`config.backend` is now always generated based on the default config. that logic needs to be in `create-config` to take user config into account

**Comment by isaachinman** [comment]
Sorry, I have been away on holiday this weekend. This does indeed sound like a genuine bug and we'll work on getting it resolved this week. Thank you for your patience.

**Comment by capellini** [comment]
@isaachinman After reviewing #140 again, perhaps c0f2a7af68baefe45f66493a006fe5ec0c57295e is a better solution than what we implemented in 56363e4e3cf7c89f55675430efe7a251a7207eaa?

**Comment by RonnyVo** [comment]
Hey guys, can you provide a config for define the url to get the translations on browser?
My scenario is I installed nextjs app in an subdirectory domain, e.g: www.example.com/my-app, the translation in server load OK but then in browser, they will load the translation json in www.example.com/static/locales instead of www.example.com/my-app/static/locales which will result in 404
